### PR TITLE
[FW] payment_mercadopago: add check payment wizard to payment provider

### PR DESCRIPTION
**Pull request by ALopez-Adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/plugberry/mercadopago/pull/60).

File: payment_mercadopago/wizards/check_payments.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    mp_amount = fields.Float()
+    mp_id = fields.Char()
+    mp_state = fields.Char()
+    mp_external_reference = fields.Char()
+    mp_partner = fields.Char()
+
+    def get_tx_info_from_mercadopago(self):
+        txt = []
+        for rec in self:
+            if rec.check_id.provider_id.provider != 'mercadopago':
```
Comment (maq-adhoc) [inline payment_mercadopago/wizards/check_payments.py]: el campo en 16 es provider_code. Por ahi te sirve usar rec.check_id.provider_id.get('provider_code')

**Review by maq-adhoc — APPROVED** [review]

**Review by maq-adhoc — APPROVED** [review]

**Comment by maq-adhoc** [comment]
@roboadhoc r+ override=ci/oba_without_tests

**Comment by roboadhoc** [comment]
I'm sorry, @maq-adhoc: you are not allowed to override this status.

**Pull request closed without merging**